src/screens/Login/index.tsx

```diff
@@ -1,7 +1,56 @@
-import React from 'react';
+import React, { useState } from 'react';
+import i18next from 'i18next';
+import { useMutation } from 'react-query';
+import { useHistory } from 'react-router-dom';
+
+import Loading from 'components/Spinner/components/loading';
+import LoginForm from 'components/Forms/LoginForm';
+import { UserCredentials } from 'utils/types';
+import { login } from 'services/UsersService';
+
+import styles from './styles.module.scss';
 
 function Login() {
-  return <div>Login component works!</div>;
+  const history = useHistory();
+  const [errorMsg, setErrorMsg] = useState('');
+  const { isLoading, isError, mutate } = useMutation((data: UserCredentials) => login(data), {
+    onSuccess: res => {
+      // eslint-disable-next-line no-console
+      console.log('access-token: ', res);
+      history.push('/home');
+    },
+    onError: () => {
+      setErrorMsg(i18next.t('FormValidations:credentials'));
+    }
+  });
+  const onValid = (userCredentials: UserCredentials) => {
+    mutate(userCredentials);
+  };
+  const onSignUpButtonClick = () => {
+    history.push('/sign_up');
+  };
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.wrapper}>
+        <div className={styles.card}>
```

Comment:
you can add the already defined classes. in the File src/scss/layout.scss

Reply:
Same as the last one commet response! 👍